src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -671,4 +673,23 @@ public TeamDetailsBundle getTeamDetailsForStudent(StudentAttributes student) {
         return null;
     }
 
+    /**
+     * Returns a {@code String} containing the duplicate email information in {@code duplicateEmailInfo} and
+     * the corresponding sanitized invalid {@code userInput}.
+     */
+    private String duplicateEmailInfo(String userInput, String duplicateEmailInfo) {
```

Comment:
Can these two methods (this and `enrollExceptionInfo`) be moved to just after `getDuplicateEmailIndex`? This is more logical as it reflects the order in which the methods are used.

Reply:
Done